source/gent/tier1_hortense.rst

```diff
@@ -12,19 +12,21 @@ General information
 **Hortense** is the 3rd VSC Tier-1 cluster, following *muk* (hosted by HPC-UGent, 2012-2016)
 and *BrENIAC* (hosted by HPC-Leuven, 2016-2022).
 
-The first phase of Hortense is also named ``dodrio``.
+The Hortense cluster also has an internal name ``dodrio``.
 
 It is available since 2021, is hosted by Ghent University,
 and maintained and supported by the HPC-UGent team.
 
+In 2023 a second phase was added, more than doubling the existing capacity of the system.
```

Comment:
In May 2023?

Reply:
previous paragraph is also vague, so i keep it the same

Reply:
May is mentioned later on. This suffices imho